.github/workflows/auto-reqs-update-pr.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           [[ ${GITHUB_REF} =~ $regex ]]
           pr_repo=$BASH_REMATCH
           # find other PRs from that repo
-          other_prs=$(curl -s ${renku_repo}/pulls | jq ".[] | select(.head.ref | test(\"auto-update/${pr_repo}\")) | .number")
```

Comment:
hmm sorry, don't get it yet: `.head.ref` will be `auto-update/renku-testing-1234`, but won't GITHUB_REF be something like `refs/pull/1493` ?

Reply:
don't think so - the event is a branch push not a PR update.

Reply:
ok, get it!